package/MDAnalysis/core/universe.py

```diff
@@ -410,7 +410,8 @@ def __init__(self, topology=None, *coordinates, all_coordinates=False,
             force_guess = list(force_guess) + ['bonds', 'angles', 'dihedrals']
 
         self.guess_TopologyAttrs(
-            context, to_guess, force_guess, vdwradii=vdwradii, **kwargs)
```

Comment:
In PR #4757 you're saying that "vdwradii should be passed to guess_TopologyAttrs or Context creation". So it's not both but just one?

Reply:
Yes so it's not applicable when guessing on Universe creation, but only when actively calling `u.guess_TopologyAttr`. It was an attempt to clarify the somewhat complex situation that all of the below are valid:

- `u.guess_TopologyAttr(DefaultGuesser(vdwradii=...))`
- `u.guess_TopologyAttr("default", vdwradii=...)`
- `u.guess_TopologyAttr(DefaultGuesser(vdwradii=...), vdwradii=...)`

I've updated #4757 to hopefully make that clearer (and that the vdwradii passed as a kwarg overrides the DefaultGuesser original setting).

Reply:
Thx